tests/ImportsNotSeparated/__snapshots__/ppsi.spec.js.snap

```diff
@@ -151,6 +151,23 @@ function add(a: number, b: number) {
 
 `;
 
+exports[`imports-with-directives.ts - typescript-verify: imports-with-directives.ts 1`] = `
```

Comment:
It would b nice to have another test case for separated imports.

Reply:
done.